packages/quantic/force-app/main/default/lwc/quanticSearchBox/quanticSearchBox.html

```diff
@@ -31,17 +31,19 @@
                 autocomplete="off"
                 role="textbox"
               />
-              <button
-                class="slds-button slds-button_icon slds-input__icon slds-input__icon_right"
-                onclick={clearInput}
-                title="Clear"
-              >
-                <lightning-icon
-                  class="slds-button__icon slds-hidden"
-                  icon-name="utility:close"
-                  size="x-small"
-                ></lightning-icon>
-              </button>
+              <template if:false={isQueryEmpty}>
+                <button
+                  class="slds-button slds-button_icon slds-input__icon slds-input__icon_right"
+                  onclick={clearInput}
+                  title="Clear"
+                >
+                  <lightning-icon
+                    class="slds-button__icon"
+                    icon-name="utility:close"
+                    size="x-small"
+                  ></lightning-icon>
+                </button>
```

Comment:
Did you try this component ?

https://developer.salesforce.com/docs/component-library/bundle/lightning-button-icon/example

Reply:
This isn't my PR :P
But no I have not. I'll try it